Backpack/src/main/java/net/skyscanner/backpack/button/BpkButton.kt

```diff
@@ -90,11 +89,10 @@ open class BpkButton(
   private val paddingVertical = resources.getDimensionPixelSize(R.dimen.bpkSpacingMd) +
     (resources.getDimensionPixelSize(R.dimen.bpkBorderSizeLg) / 2)
 
-  final override var icon: Drawable? = null
-    set(value) {
-      field = value
-      updateIconState()
-    }
+  override fun setIcon(icon: Drawable?) {
```

Comment:
that's from `MaterialButton`

Reply:
This may potentially be breaking change as we replace property with setter. It may not be automatically converted to property, let's just double-check the compatibility for Kotlin here.

Reply:
yep this works - we actually use the property within the button component as well (e.g. in the constructor)